tests/lib/test_generate_commit_message.py

```diff
@@ -34,9 +35,8 @@ def test_Repo_generate_commit_message(repo: Repo) -> None:
     to the root of the repository.
     """
     # modify the repository with some different commands:
-    repo.mkdir(repo.sanitize_path("a/new/folder"))
-    repo.mv(repo.sanitize_path("s p a c e s"),
-            repo.sanitize_path("a/new/folder"))
+    repo.mkdir(Path("a/new/folder").resolve())
+    repo.mv(Path("s p a c e s").resolve(), Path("a/new/folder").resolve())
```

Comment:
For the purpose of the test we know the path exactly: `repo.root / 'a' / 'new' / 'folder'`. No need to resolve anything. Resolving also is actually wrong, since this is resolving a relative path against CWD. But CWD is irrelevant here - we don't want to hardcode assumptions about where the tests are running from. All that matters here is that we pass absolute paths to`Repo.mv`.